GVRf/Framework/src/org/gearvrf/GVRScene.java

```diff
@@ -212,8 +213,21 @@ void updateStats() {
 
             mStatsConsole.writeLine("Draw Calls: %d", numberDrawCalls);
             mStatsConsole.writeLine(" Triangles: %d", numberTriangles);
+            
+            if(mStatMessage!="")
+            	mStatsConsole.writeLine("%s", mStatMessage);
```

Comment:
Agreed. Probably you may need to reset mStatMessage within some time. It is not like the draw calls and triangles messages while they are updating their message every frame.